packages/external-db-config/lib/readers/aws_config_reader.js

```diff
@@ -23,77 +21,85 @@ class AwsConfigReader {
     return JSON.parse(data.SecretString)
   }
 
+  async getExternalAndLocalEnvs() { 
+    const externalConfig = await this.readExternalConfig().catch(emptyConfig)
```

Comment:
It will be much clearer, if `readExternalConfig` will **always** return an object with the next 2 properties:  `externalConfig` and `secretMangerError`

Reply:
Therefore, it will be necessary to move the scenario in which the reading fails into the function